angular-hotkeys/angular-hotkeys.d.ts

```diff
@@ -9,13 +9,16 @@ declare module ng.hotkeys {
 
```

Comment:
sry.. even forgot to push a commit yesterday.. [_del overload](https://github.com/reppners/DefinitelyTyped/commit/57708d0b2416820cb7946a3ffb467d32fa1616fb)
but this one is merely an overload for del() which accepts a hotkey object.